trade-finance-manager-ui/server/controllers/helpers/validateSubmittedValues.helper.test.js

```diff
@@ -5,41 +5,18 @@ import generateValidationErrors from '../../helpers/validation';
 
 describe('POST underwriting - managers decision - validate submitted values', () => {
   describe('validateCommentField', () => {
-    describe('when field is NOT alphanumeric', () => {
```

Comment:
Can we please add following test assertions:

1. Lorem ipsum with special characters previously not allowed.
2. An empty input with white spaces `   `
3. An input with various HTML tags

Reply:
What's the expected behaviour for the second case? I think previously whitespace was allowed.

It's not a required field, but it might be confusing if a user does save a whitespace response. So maybe it makes most sense to trim any leading/trailing whitespace, but not make a validation error for it. What do you think?

Reply:
Oh actually, the validator doesn't change the value at all so it's not simple to trim any whitespace. I think if we do want to add that behaviour we should raise a separate ticket, but if not the behaviour is unchanged so it's probably not worth adding a test for it here.

For assertion 3, the HTML tags are stripped out by the middleware rather than this method so I won't add a test with them here.

Reply:
2. To ensure an empty input made of white spaces only is not saved and a validation error is thrown.
3. Understood, if we can cover this in an E2E test then that will suffice.

Reply:
Oh I see, I missed that there's already some validation to disallow whitespace. 

That validation doesn't happen in this file so I've not added a test here, but I've added a test for whitespace in `underwriter-managers-decision.spec.js`.

Same for the HTML tags, I've added a check for it in that file.